app/src/main/res/values/strings.xml

```diff
@@ -114,25 +114,25 @@
     <string name="toast_save_to">Save file to: </string>
     <string name="toast_operation_failed">The operation failed.</string>
     <string name="toast_operation_done">The operation done.</string>
-    <string name="volume">Press Volumekey to Scroll</string>
-    <string name="intent">Allow Website Redirect to App</string>
-    <string name="msg_intent">Would like to open your app?</string>
-    <string name="dialog_adblock">Default Block:Use Local hosts.txt to Block Ads\n\nCustom Block:Use storage hosts.txt, it will download a file if unexisted.</string>
+    <string name="volume">Press Volume Keys to Scroll</string>
+    <string name="intent">Allow Website to Run Apps</string>
```

Comment:
Maybe you could change this to Allow This Website to Open its App

Reply:
I don’t think the app is developed by or has anything to do with the website, so it’s not “its App”.